examples/streaming-examples.ts

```diff
@@ -0,0 +1,43 @@
+import {AudioConfig, RevAiStreamingClient} from 'revai-node-sdk';
+import * as fs from 'fs';
+
+/* Initialize your client with your audio configuration and access token */
+const audioConfig = new AudioConfig(
+	/* contentType */ "audio/raw", 
+	/* layout */      "interleaved", 
+	/* sample rate */ 16000,
+	/* format */      "S16LE",
+    /* channels */    1
```

Comment:
Formatting looks weird. Not sure if it's just github though

Reply:
I think its a combination of tabs and spaces, files have been adjusted